Assets/Scenes/TutorialLevel.unity

```diff
@@ -16678,17 +16599,6 @@ Transform:
   m_CorrespondingSourceObject: {fileID: 7930691839557344146, guid: fe9a12e8b29f2cb5ba644d961fd951ce, type: 3}
   m_PrefabInstance: {fileID: 1621152295}
   m_PrefabAsset: {fileID: 0}
---- !u!114 &1820880696 stripped
-MonoBehaviour:
```

Comment:
What is this deletion?

Reply:
The second Adam being deleted as discussed.